includes/Modules/Analytics.php

```diff
@@ -282,15 +282,25 @@ protected function handle_provisioning_callback() {
 			exit;
 		}
 
+		$internal_web_property_id = $web_property->getInternalWebPropertyId();
+
 		$this->get_settings()->merge(
 			array(
 				'accountID'             => $account_id,
 				'propertyID'            => $web_property_id,
 				'profileID'             => $profile_id,
-				'internalWebPropertyID' => $web_property->getInternalWebPropertyId(),
+				'internalWebPropertyID' => $internal_web_property_id,
 			)
 		);
 
+		do_action(
+			'googlesitekit_analytics_handle_provisioning_callback',
+			$account_id,
+			$web_property_id,
+			$internal_web_property_id,
+			$profile_id
+		);
```

Comment:
Let's expect this to be called in the `AnalyticsTest::test_handle_provisioning_callback` as well.

Reply:
Added.